src/components/SettingsButton.jsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
```

Comment:
Aqui, na importação do react, é legal usar sempre aspas simples para o ESLint não acusar problema.
```suggestion
import React from 'react';
```